provider/cmd/pulumi-resource-aws-apigateway/provider.ts

```diff
@@ -43,6 +43,10 @@ async function constructRestAPI(name: string, inputs: pulumi.Inputs,
         urn: restAPI.urn,
         state: {
             url: restAPI.url,
+            api: restAPI.api,
+            deployment: restAPI.deployment,
+            stage: restAPI.stage,
+            apiPolicy: restAPI.apiPolicy,
```

Comment:
Not actually sure if we need this change to the state as well as the `registerOutputs`?